### DISCUSSION TITLE
Setting album cover freaks out the image grid

### DISCUSSION THREAD

**Discussion by Puupuls** [LycheeOrg/Lychee#2359] [Bugs] [answered]
### Lychee version

v5.1.2

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

Uploading Screen Recording 2024-04-08 at 23.08.08.mov…

### Steps to reproduce the issue

1) Open album
2) Right click on image
3) Set as cover image
4) Everything breaks

### Diagnostics [REQUIRED]

    Diagnostics
    -----------
    Warning: /v**********************lic/uploads/ has permissions 2775, but should have 2770
    Warning: /v**********************lic/uploads/tracks has permissions 2775, but should have 2770
    Warning: /v**********************lic/uploads/original has permissions 2775, but should have 2770
    Warning: /v**********************lic/uploads/original/82 has permissions 2775, but should have 2770
    Warning: /v**********************lic/uploads/original/82/2a has permissions 2775, but should have 2770
    Warning: 93 more directories with wrong permissions
    Warning: Dropbox import not working. dropbox_key is empty.
    Warning: lossless_optimization set to 1 but svgo not found!
    Info: Latest version of PHP is 8.3
    Warning: Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
    Warning: Online updates are disabled by configuration

    System Information
    ------------------
    Lychee Version (tag):                    v5.2.0 (31a394b) -- Up to date (11 hours ago).
    DB Version:                              5.2.0
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               false
    APP_URL:                                 set
    APP_DIR:                                 default
    LOG_VIEWER_ENABLED:                      true
    LIVEWIRE_ENABLED:                        true
    
    System:                                  Linux
    PHP Version:                             8.2.7
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                UTC
    Max uploaded file size:                  100M
    Max post size:                           100M
    Livewire chunk size:                     12.00 MB
    Max execution time:                      3600
    MySQL Version:                           10.11.7-MariaDB-1:10.11.7+maria~ubu2204
    
    exec() Available:                        yes
    Imagick Available:                       1
    Imagick Enabled:                         1
    Imagick Version:                         1691
    GD Version:                              2.3.3
    Number of foreign key:                   12 found.

    Config Information
    ------------------
    version:                                 050200
    check_for_updates:                       1
    sorting_photos_col:                      taken_at
    sorting_photos_order:                    ASC
    sorting_albums_col:                      max_taken_at
    sorting_albums_order:                    DESC
    imagick:                                 1
    skip_duplicates:                         1
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    en
    image_overlay_type:                      desc
    default_license:                         none
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     0
    site_owner:                              Pauls Purviņš (Puupuls)
    landing_title:                           Pauls Purviņš
    landing_subtitle:                        Cats, Dogs & Humans Photography
    sm_facebook_url:                         https://www.facebook.com/puupuls/
    sm_flickr_url:                           
    sm_twitter_url:                          
    sm_instagram_url:                        https://www.instagram.com/puupuls/
    sm_youtube_url:                          https://www.youtube.com/@puupuls
    landing_background:                      dist/cat.webp
    site_title:                              Puupuls Photos
    footer_show_copyright:                   1
    site_copyright_begin:                    2020
    site_copyright_end:                      2024
    footer_additional_text:                  
    footer_show_social_media:                0
    search_public:                           0
    SL_enable:                               0
    SL_for_admin:                            0
    recent_age:                              1
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             0
    zip64:                                   1
    map_display_public:                      0
    map_provider:                            Wikimedia
    force_32bit_ids:                         0
    map_include_subalbums:                   0
    update_check_every_days:                 3
    has_exiftool:                            1
    share_button_visible:                    1
    import_via_symlink:                      0
    has_ffmpeg:                              1
    location_decoding:                       0
    location_decoding_timeout:               30
    location_show:                           1
    location_show_public:                    0
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   1
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            1
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 0
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    SA_enabled:                              0
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row
    allow_username_change:                   0
    auto_fix_orientation:                    1
    use_job_queues:                          0
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 0
    ffmpeg_path:                             /usr/bin/ffmpeg
    ffprobe_path:                            /usr/bin/ffprobe
    layout:                                  masonry
    date_format_photo_thumb:                 Y.m.d, G:i:s e
    date_format_photo_overlay:               Y.m.d, G:i:s e
    date_format_sidebar_uploaded:            Y.m.d, G:i:s e
    date_format_sidebar_taken_at:            Y.m.d, G:i:s e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             Y.m.d, G:i:s T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover
    default_album_thumb_aspect_ratio:        1/1
    use_album_compact_header:                0
    login_button_position:                   left
    back_button_enabled:                     0
    back_button_text:                        Return to Home
    back_button_url:                         /
    current_job_processing_visible:          1

### Browser & System [REQUIRED]

Chrome: Version 123.0.6312.107 (Official Build) (arm64)
MacOS: Sonoma Version 14.4.1 (23E224)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Yeah I know, already opened: https://github.com/LycheeOrg/Lychee/issues/2194 
I haven't had time to fix it.
It is not so high on my priority list as it is more of a "happens once in a while" kind of bug and it does not hinder Lychee behaviour too much (but yes it is annoying).
